### PR TITLE
Organize cts source directory better

### PR DESCRIPTION
**Pull request by kgaillot**
This moves files related to cts-support into a new cts/support/ directory, and files related to the original CTS into a new cts/lab/ directory. The installed locations do not change.

The main motivation is to be able to search related code more easily, e.g. grep whatever cts/lab/*. It's also less confusing when you need to work on it.

I initially wanted to move files related to regression tests into a cts/regression/ directory as well, but given that it would involve thousands of files, and CI/specfiles/etc. might be running them from current locations, I decided against it.

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this when you get a chance?

@wenningerk @gao-yan , heads-up for files moving around. Let me know if you think it's a bad idea.

**Comment by wenningerk** [comment]
> @wenningerk @gao-yan , heads-up for files moving around. Let me know if you think it's a bad idea.

Having a representation of the difference between cts-regression-test & cts-lab in the source-tree seems like a good idea to me.

File: cts/Makefile.am [hunk 1]
```diff
@@ -85,10 +76,12 @@ CLEANFILES	= $(builddir)/.regression.failed.diff
 clean-local:
 	rm -f scheduler/*/*.pe
 
-SUBDIRS	= benchmark scheduler
+SUBDIRS	= benchmark scheduler support
 
-cts-support-install: cts-support
-	./cts-support install
+cts-support-install:
+	make -C support cts-support
```
Comment (clumens) [inline cts/Makefile.am:56]: I've been doing a lot of stuff with make at home recently, and you should apparently always use $(MAKE) -C for things like this (and the next target down).

Comment (kgaillot) [reply cts/Makefile.am:56]: Oh, right, $(MAKE) $(AM_MAKEFLAGS) actually, I should have remembered that as I once made sure everything was using it :)

**Comment by clumens** [comment]
Everything else looks good to me.

File: cts/Makefile.am (hunk 1 above)
Comment (kgaillot) [reply cts/Makefile.am]: Fixed (and rebased on current master)

**Comment by gao-yan** [comment]
Missed this one, sorry. Yes, good idea!

**Pull request merged**